doc/design/ROS_COMPARISON.md

```diff
@@ -16,13 +16,9 @@ In addition, move_base itself has been split into multiple components:
 
 ![Move Base 2](./move_base_compare_2.png)
 
-The *nav2_bt_navigator* replaces move_base at the top level, with a *Task* interface to call the global and local planners with a configurable tree-based action model.
+The *nav2_bt_navigator* replaces move_base at the top level, with an *Action* interface to call the global planner and controller with a configurable tree-based action model.
```

Comment:
since you are changing the local planner to be now controller, should we have a follow-up for dropping "global" and leave it as just "planner"?

Reply:
Global planner still accurately describes the task since its not a local planner. Er, um by ROS conventions throw out what you think of as a "local planner".

Think of the global planner --> route planning
Think of local planner --> planning
Think of controller --> controller